.circleci/config.yml

```diff
@@ -4,26 +4,6 @@ jobs:
       docker:
         - image: circleci/node:8.11
         
-        - image: postgres:9.5-alpine
```

Comment:
Why we are removing all the integration tests. They can fail on CI until we fix it.